native/ax/Native2.0/NativeCore/NativeMemory.c

```diff
@@ -303,6 +303,8 @@ char* sagetv_mem_inf( int loc, char *buf, int buf_size )
 }
 
 #else
+

```

Comment:
@qianzhang5 - can you review this change and confirm the decision to disable the code here, while leaving the like-named fn in TSParser.c.  Note that the behavior of the two like-named fns aren't **exactly** identical.  They especially differ w/respect to the test for requested space to be malloc'd.  Thanks